frontend/packages/operator-lifecycle-manager/src/components/subscription.tsx

```diff
@@ -341,7 +341,9 @@ export const SubscriptionDetails: React.FC<SubscriptionDetailsProps> = ({
   const installFailedMessage =
     installFailedCondition?.message || installFailedCondition?.reason || 'Install plan failed';
 
-  const pkg = packageForSubscription(packageManifests, obj);
+  const pkg = _.isArray(packageManifests)
+    ? packageForSubscription(packageManifests, obj)
+    : packageManifests;
```

Comment:
We can remove this block if we add a similar wrapper for `SubscriptionDetailsPage` at https://github.com/openshift/console/blob/master/frontend/packages/operator-lifecycle-manager/src/components/subscription.tsx#L594

Reply:
I don't think it's a good idea to overload the prop to have two different types. It would be better to do it one way in all contexts.